tools/editor/scenes_dock.cpp

```diff
@@ -959,6 +960,17 @@ void ScenesDock::_file_option(int p_option) {
 
 
 			String path = files->get_item_metadata(idx);
+			if (p_option == FILE_SHOW_IN_EXPLORER) {
+			  String dir = DirAccess::get_full_path("res://", DirAccess::ACCESS_FILESYSTEM);
+				const int res_begin = String("res://").length();
+				const int last_sep = path.find_last("/");
+				if (last_sep > res_begin) {
+					dir += "/";
+					dir += path.substr(res_begin, last_sep - res_begin);
+				}
+				OS::get_singleton()->shell_open(String("file://")+dir);
```

Comment:
Hm that's weird, it's supposed to return ``get_current_dir()`` (https://github.com/godotengine/godot/blob/a895e2e372c9df2a9972f38134ae0ea9c8bf45a8/core/os/dir_access.cpp#L357-L367) which does basically what you did manually here:
- https://github.com/godotengine/godot/blob/a895e2e372c9df2a9972f38134ae0ea9c8bf45a8/drivers/windows/dir_access_windows.cpp#L225-L242
- https://github.com/godotengine/godot/blob/a895e2e372c9df2a9972f38134ae0ea9c8bf45a8/drivers/unix/dir_access_unix.cpp#L264-L277

It sounds like a bug in DirAccess if it doesn't.